test/integration/testdata/gen-config-then-flags.golden

```diff
@@ -96,8 +96,8 @@ data:
       - name: E2E_USE_GO_RUNNER
         value: "true"
       - name: SONOBUOY_K8S_VERSION
-        value: v1.20.0
-      image: k8s.gcr.io/conformance:v1.20.0
+        value: ignore
+      image: k8s.gcr.io/conformance:ignore
```

Comment:
The ignore mode is just for this sort of thing; when doing `gen` locally and you don't have a cluster to find. Rather than erroring out it just swallows those errors and sets this string to unblock things. Good for testing and hacking.